src/multi-select/tp-multi-select-pill.ts

```diff
@@ -13,16 +13,24 @@ export class TPMultiSelectPillElement extends HTMLElement {
 	connectedCallback(): void {
 		this.querySelector( 'button' )?.addEventListener( 'click', this.handleButtonClick.bind( this ) );
 	}
-
 	/**
 	 * Handle button click.
 	 *
-	 * @param {Event} e Click event.
+	 * @param {any} e Click event.
 	 */
-	handleButtonClick( e: Event | null ): void {
+	handleButtonClick( e: any | null ): void {
```

Comment:
@imranhsayed I think the problem is the fact that this button does not have a type `button`? https://github.com/Travelopia/web-components/blob/master/src/multi-select/tp-multi-select-pills.ts#L70

Changing that to `<button type="button">x</button>` would solve it?

Reply:
That was my first try @junaidbhura 
That didn't work.